src/content/recipes/@vanilla-extract/css.md

```diff
@@ -52,10 +45,17 @@ npx storybook@latest add @storybook/addon-styling-webpack
 
 This will run a configuration script that will walk you through setting up the addon. If prompted, select `Vanilla Extract` as your CSS processor.
 
-### Manually configure Webpack
+<details>
+  <summary>Did the configuration script fail?</summary>
+  <p>Under the hood, this command runs <code>npx @storybook/auto-config styling</code>, which is responsible for reading your project and attempting to configure your Storybook Webpack for your desired tools. If running that command directly does not resolve your issue, please consider filing a bug report on the <a href="https://github.com/storybookjs/auto-config/issues/new?assignees=&labels=bug&projects=&template=bug_report.md&title=%5BBug%5D" target="_blank">@storybook/auto-config</a> repository so that we can further improve it. For manual configuration instructions for Less, you can refer to the documentation <a href="https://github.com/storybookjs/addon-styling-webpack" target="_blank">here</a>.</p>
```

Comment:
Best to avoid linking just the word "here".

```suggestion
  <p>Under the hood, this command runs <code>npx @storybook/auto-config styling</code>, which is responsible for reading your project and attempting to configure your Storybook Webpack for your desired tools. If running that command directly does not resolve your issue, please consider filing a bug report on the <a href="https://github.com/storybookjs/auto-config/issues/new?assignees=&labels=bug&projects=&template=bug_report.md&title=%5BBug%5D" target="_blank">@storybook/auto-config</a> repository so that we can further improve it. For manual configuration instructions for Less, you can refer to the <a href="https://github.com/storybookjs/addon-styling-webpack" target="_blank">documentation</a>.</p>
```

Reply:
Also, I like the "so we can further improve it" wording better than the other similar passages in other recipes. If if it's too much trouble, consider making that consistent?

Reply:
For sure, I thought I had copied it over to be consistent already. but I'll double check